app/controllers/placements/providers/placements_controller.rb

```diff
@@ -5,8 +5,7 @@ class Placements::Providers::PlacementsController < ApplicationController
   def index
     @subjects = Subject.order_by_name.select(:id, :name)
     @school_types = compact_school_attribute_values(:type_of_establishment)
-    @schools = Placements::School.select(:id, :name)
-    @partner_schools = @provider.partner_schools.select(:id, :name)
+    set_schools
```

Comment:
...and then this becomes

```suggestion
    @schools = schools_scope.select(:id, :name)
```